eth/downloader/resultstore.go

```diff
@@ -141,7 +141,7 @@ func (r *resultStore) HasCompletedItems() bool {
 // countCompleted returns the number of items ready for delivery, stopping at
 // the first non-complete item.
 //
-// The mthod assumes (at least) rlock is held.
+// The method assumes (at least) r.lock is held.
```

Comment:
I think this comment was originally intending to say that the read lock was held for `countCompleted` because it says "at least". So now you've changed it to say that the lock is held, was that intentional?

Reply:
Hey @palango Are you saying the change is intentional or not?

Reply:
I thought this referred to the field in the struct, but your intepretation makes more sense, so I'm gonna revert this.